config/redirects.txt

```diff
@@ -8,19 +8,21 @@
 (.*)\.md(.*) -> $1$2
 ^\/introduction\/introduction(\/|$) -> /introduction
 ^\/$ -> /introduction
-^\/installing\/gettingStarted(\/|$) -> /raspberrypi/nodejs/getting-started/
+^\/installing\/gettingStarted(\/|$) -> /raspberrypi3/nodejs/getting-started/
 ^\/installing\/gettingStarted-BBB(\/|$) -> /beaglebone/nodejs/getting-started/
 ^\/installing\/gettingStarted-Humming(\/|$) -> /hummingboard/nodejs/getting-started/
 ^\/installing\/gettingStarted-NUC(\/|$) -> /nuc/nodejs/getting-started/
 ^\/installing\/gettingStarted-Nitrogen6x(\/|$) -> /nitrogen6x/nodejs/getting-started/
 
 
 # Default redirects
-^\/getting-started -> /raspberrypi3/nodejs/getting-started
-^\/deployment\/network\/(2\.0.*|$) -> /deployment/network/2.x
-^\/deployment\/network\/1.x.x -> /deployment/network/1.x
-^\/understanding/understanding-devices\/(2\.0.*|$) -> /understanding/understanding-devices/2.x
-^\/understanding/understanding-devices\/1.x.x -> /understanding/undderstanding-devices/1.x
+^\/getting-started($) -> /raspberrypi3/nodejs/getting-started/
+^\/deployment\/network\/(2\.0.*|$) -> /2.x/deployment/network
+^\/deployment\/network\/1.x.x -> /1.x/deployment/network
```

Comment:
Is this right? The URLs I see for this right now are `https://docs.resin.io/deployment/network/1.x/` and `https://docs.resin.io/deployment/network/2.x/`, and this doesn't match either. I suspect the same is true below too.

Reply:
@pimterry that's an older redirect for an even earlier version of the pages that had the versions as 2.0.0 and 1.x.x.